tools/setup_helpers/gen_unboxing.py

```diff
@@ -0,0 +1,11 @@
+# Little stub file to get BUILD.bazel to play along
+
+import os.path
+import sys
+
+root = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+sys.path.insert(0, root)
+
+import tools.jit.gen_unboxing
+
+tools.jit.gen_unboxing.main()
```

Comment:
@wconstab has been adding new codegen recently, and he asked me about whether or not he should make a new entrypoint or not. I said that, absent other considerations, it's easier to reuse the existing entrypoint. So I'm wondering if there is a good reason to make a new endpoint for unboxing codegen.

Reply:
One justification I can think of is by having a separate endpoint, you can control whether or not you run the codegen or not. But for the sake of debugging, it's far easier if we unconditionally generate all the files in all builds (and then make choices about whether or not to build them or not). Also, if you REALLY wanted, you could still have a flag to toggle whether or not the codegen happened or not.

Reply:
@bdhirsh suggested that this codegen to be completely isolated from `gen.py`. I'm fine with either way.

Reply:
Oh, that was mostly my suggestion, mostly for two reasons:

(1) I figured that if mobile is the only consumer of the codegen'd unboxing, then it would be easier to reason about it we keep it in a separate script instead of cluttering up `gen.py` with another flag.

(2) The mobile codegen generates unboxing kernels under the `aten` namespace, but it also generates registrations to the `jit` registry, which live under the `torch` namespace. At least right now, I think that `gen.py` only generates stuff that lives in `aten` (and for example, the autograd codegen has a separate entrypoint. Also I guess that won't be true anymore if/once LTC goes directly in `gen.py`).

Reply:
Thanks for looping me in @ezyang.  My approach (still WIP and only tested via buck so far) is D34333432 if you want to compare.  I think it ends up being pretty convenient to tie into gen.py but I may run into issues as I try to finish it up.